<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExtremeValues</class>
 <widget class="QWidget" name="DialogExtremeValues">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extreme Values</string>
  </property>
  <property name="windowIcon">
   <iconset>icons:extremes.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QComboBox" name="comboParam"/>
     </item>
     <item>
      <widget class="MiDateTimeEdit" name="timeFrom">
       <property name="displayFormat">
        <string notr="true">yyyy-MM-dd hh:mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTo">
       <property name="text">
        <string> to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MiDateTimeEdit" name="timeTo">
       <property name="displayFormat">
        <string notr="true">yyyy-MM-dd hh:mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableExtremes"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MiDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>util/MiDateTimeEdit.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonApply</sender>
   <signal>clicked()</signal>
   <receiver>DialogExtremeValues</receiver>
   <slot>onUpdateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onUpdateClicked()</slot>
 </slots>
</ui>
